public/javascript/play.js

```diff
@@ -4,37 +4,41 @@ var makeRequest = (content) => {
     if (!httpRequest) {
         alert('Giving up :( Cannot create an XMLHTTP instance');
         return false;
+    };
+    httpRequest.onreadystatechange = () => {
+        if (httpRequest.readyState === 4 && httpRequest.status === 200) {
+            console.log(httpRequest.responseText);
+        }
+        httpRequest.open('POST', '/bets');
+        httpRequest.setRequestHeader('Content-Type', 'application/json');
+        httpRequest.send(JSON.stringify(content));
     }
```

Comment:
This won't work if the request doesn't get sent.
```suggestion
    httpRequest.onreadystatechange = () => {
        if (httpRequest.readyState === 4 && httpRequest.status === 200) {
            console.log(httpRequest.responseText);
        }
    }
    httpRequest.open('POST', '/bets');
    httpRequest.setRequestHeader('Content-Type', 'application/json');
    httpRequest.send(JSON.stringify(content));
```
The instructions to open and send the request need to be outside the `onreadystatechange` method.